test/functions.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { readJSON } from 'fs-extra'
+import mock from 'mock-fs'
```

Comment:
Missing `mock-fs` from the dev dependencies in `package.json`?

Reply:
Yeah, just realized that. I’ll push it up in a bit.